src/scenes/Event/components/Row.js

```diff
@@ -1,13 +1,10 @@
 import React from 'react';
-import styled from 'styled-components';
-import { Grid, Paper } from '@material-ui/core';
+import { Grid, Paper, withStyles } from '@material-ui/core';
 
+import styles from './styles';
 
-export const Row = styled(({ children, ...props }) => (
-  <Paper {...props}>
+export const Row = withStyles(styles)(({ children, ...props }) => (
+  <Paper elevation={0} {...props}>
```

Comment:
Here comes bug by using `...props` here. All `classes` data from `./styles` will fill into Paper's field, which result in style related error in console.

Reply:
Good catch. I have removed this. Doesn't seem like anything is being passed down props-wise.